pyscf/cc/ccsd.py

```diff
@@ -1126,7 +1127,9 @@ def ccsd_t(self, t1=None, t2=None, eris=None):
         from pyscf.cc import ccsd_t
         if t1 is None: t1 = self.t1
         if t2 is None: t2 = self.t2
-        if eris is None: eris = self.ao2mo(self.mo_coeff)
+        if eris is None:
+            eris = self.ao2mo(self.mo_coeff)
+            self.e_hf = self.get_e_hf(mo_coeff=self.mo_coeff)
```

Comment:
self.e_hf may not properly initialized if eris is specified in input

Reply:
I've deleted the self.e_hf evaluation here.